jans-cedarling/bindings/cedarling_python/tests/test_logger.py

```diff
@@ -4,26 +4,24 @@
 # Copyright (c) 2024, Gluu, Inc.
 
 from cedarling_python import Cedarling
-from config import sample_bootstrap_config
+from config import load_bootstrap_config
 
 
 # In python unit tests we not cover all possible scenarios, but most common.
 
-def test_invalid_log_config(sample_bootstrap_config):
-    # map fixture to variable with shorter name for readability
-    config = sample_bootstrap_config
+def test_invalid_log_config(load_bootstrap_config):
```

Comment:
Looks like it has no sense to use fixture with closure

Reply:
fixed here: [1dcc381](https://github.com/JanssenProject/jans/pull/10350/commits/1dcc381ed868dc8e5d98b472cf2d657cd06b7fa5)